### PR TITLE
Create 2024 keynote bios blog post

### PR DESCRIPTION
**Pull request by AlfredoMS**
Blog post announcing keynote speaker for 2024

### CODE REVIEW

**Comment by altsalt** [comment]
Let's sort alphabetically, no strong preference on first or last name.

Do we have headshots, if so could we include them?

File: _posts/2024-09-29-Keynotes-intro.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+layout: post
+title: 'Announcing Keynote speakers for SeaGL2024!'
+status: publish
+type: post
+published: true
+categories: news
+tags: '2024'
+---
+
+We are very pleased to announce our 2024 Seattle GNU/Linux conference keynote speakers:
```
Comment (AndrewKvalheim) [inline _posts/2024-09-29-Keynotes-intro.md:11]:
```suggestion
We are very pleased to announce our 2024 Seattle GNU/Linux conference keynote speakers:
```

File: _posts/2024-09-29-Keynotes-intro.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## Christopher Neugebauer
+
+Australian developer, speaker, and serial community conference organizer, who presently lives in the United States.
+
+He serves as a Director of the Python Software Foundation, and is co-organizer of the acclaimed North Bay Python conference, a boutique one-track conference run in unusual venues — include an old vaudeville theater, and more recently a barn on a farm — in Petaluma, California.
+
+Christopher is also a contributor on the open source Pants build system, helping make Python’s testing, correctness, and style tools accessible and fast for developers, no matter how big their codebase.
+
+Find more about Christopher on:
+ - Website: https://chrisjrn.com/
```
Comment (AndrewKvalheim) [inline _posts/2024-09-29-Keynotes-intro.md:22]:
```suggestion
 - Website: <https://chrisjrn.com/>
```

File: _posts/2024-09-29-Keynotes-intro.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ - Fediverse/Mastodon: [@chrisjrn@social.coop](https://social.coop/@chrisjrn)
+ - Twitter/X: [@chrisjrn](https://x.com/chrisjrn)
+
+## Duane O'Brien
+
+Duane is the Director of Collaborative Engineering at Capital One. His organization focuses on developing internal and external collaborative development practices through open source and InnerSource. He is the creator of the [FOSS Contributor Fund framework](https://github.com/indeedeng/FOSS-Contributor-Fund), and loves helping organizations get involved in funding and sustaining their open source dependencies. Duane is a force of chaotic good using his high stats in intelligence and charisma to advocate for the open source community. If you encounter him in forested areas, he will share his fire, drink, and philosophy.
+
+Find more about Duane on:
+ - Website: https://www.duaneobrien.com/
```
Comment (AndrewKvalheim) [inline _posts/2024-09-29-Keynotes-intro.md:31]:
```suggestion
 - Website: <https://www.duaneobrien.com/>
```

File: _posts/2024-09-29-Keynotes-intro.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ - Fediverse/Mastodon: [@duane@hachyderm.io](https://hachyderm.io/@duane)
+
+## Aaron Wolf
+
+First time keynote speaker, Aaron is a community music teacher, co-founder of [Snowdrift.coop](https://snowdrift.coop/) (a long-struggling and principled platform working to solve economic coordination dilemmas around FLO public goods), and an activist and volunteer in many other areas. Originally from Ann Arbor, MI; he lives now in Oregon City with his wife, dog, and two kids.
```
Comment (AndrewKvalheim) [inline _posts/2024-09-29-Keynotes-intro.md:36]:
```suggestion
First time keynote speaker, Aaron is a community music teacher, co-founder of [Snowdrift.coop](https://snowdrift.coop/) (a long-struggling and principled platform working to solve economic coordination dilemmas around FLO public goods), and an activist and volunteer in many other areas. Originally from Ann Arbor, MI; he lives now in Oregon City with his wife, dog, and two kids.
```

File: _posts/2024-09-29-Keynotes-intro.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+## Aaron Wolf
+
+First time keynote speaker, Aaron is a community music teacher, co-founder of [Snowdrift.coop](https://snowdrift.coop/) (a long-struggling and principled platform working to solve economic coordination dilemmas around FLO public goods), and an activist and volunteer in many other areas. Originally from Ann Arbor, MI; he lives now in Oregon City with his wife, dog, and two kids. 
+
+Find more about Aaron on:
+ - Fediverse/Mastodon: [@wolftune@social.coop](https://social.coop/@wolftune)
+
+## Rachel Kelly
+
+Another first time keynote speaker and a senior site reliability engineer in Portland, OR.
```
Comment (AndrewKvalheim) [inline _posts/2024-09-29-Keynotes-intro.md:43]:
```suggestion
Another first time keynote speaker and a senior site reliability engineer in Portland, OR.
```

File: _posts/2024-09-29-Keynotes-intro.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+First time keynote speaker, Aaron is a community music teacher, co-founder of [Snowdrift.coop](https://snowdrift.coop/) (a long-struggling and principled platform working to solve economic coordination dilemmas around FLO public goods), and an activist and volunteer in many other areas. Originally from Ann Arbor, MI; he lives now in Oregon City with his wife, dog, and two kids. 
+
+Find more about Aaron on:
+ - Fediverse/Mastodon: [@wolftune@social.coop](https://social.coop/@wolftune)
+
+## Rachel Kelly
+
+Another first time keynote speaker and a senior site reliability engineer in Portland, OR. 
+One of the organizers of the Portland chapter of PyLadies, an international group of women Python coders, and also chaired (with a group of six) a small Python conference here in Portland called PyDX, which has morphed into the wonderful PyCascadia.  Past contributor and volunteer to SeaGL in 2020 and probably 2021.
```
Comment (AndrewKvalheim) [inline _posts/2024-09-29-Keynotes-intro.md:44]:
```suggestion
One of the organizers of the Portland chapter of PyLadies, an international group of women Python coders, and also chaired (with a group of six) a small Python conference here in Portland called PyDX, which has morphed into the wonderful PyCascadia. Past contributor and volunteer to SeaGL in 2020 and probably 2021.
```

(non-breaking space)

File: _posts/2024-09-29-Keynotes-intro.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+Find more about Rachel on:
+ - Website: https://rkode.com/
```
Comment (AndrewKvalheim) [inline _posts/2024-09-29-Keynotes-intro.md:47]:
```suggestion
 - Website: <https://rkode.com/>
```

**Review by AndrewKvalheim — COMMENTED** [review]
Just some Markdown suggestions

File: _posts/2024-09-29-Keynotes-intro.md (hunk 4 above)
Comment (polygonart6777) [reply _posts/2024-09-29-Keynotes-intro.md:36]: This is knit-picky and possibly not necessary, but should the wording be "he now lives in Oregon City..."

**Pull request merged**